tools/log-collector/log_collector.go

```diff
@@ -734,17 +509,92 @@ func (l *LogCollector) CheckIsOpenshift() bool {
 
 // getResourceObjectsWithOwnerRef return all the objects which has ownerRef of CSV
 func (l *LogCollector) getResourceObjectsWithOwnerRef(resourcePath string,
```

Comment:
rename func to `getOcpResourcesByOwnerRef` or somthing better

Reply:
resolved